src/input/mod.rs

```diff
@@ -2035,199 +2045,202 @@ impl State {
                     _ => unreachable!(),
                 };
 
-                let current_output = seat.active_output();
-                let mut shell = self.common.shell.write().unwrap();
-                let next_output = shell.next_output(&current_output, direction).cloned();
+                if let Some(focused_output) = seat.focused_output() {
```

Comment:
Also instead of wrapping everything in an `if` we could just do something like:
`let Some(focused_output) = seat.focused_output() else { return };`

These kind of actions don't do anything without focus anyway.

Reply:
do you want me to change the existing instances or just do that going forward?

Reply:
I would like you to change the existing instances as well for consistency and readability. The extra level of indentation also makes the git-diff far more annoying to parse.

Reply:
implemented